### ISSUE TITLE
New property gets a min/max value of infinite or some really big number when added to MustardUI

### ISSUE THREAD

**Issue by J4ke6599** [Mustard2/MustardUI#186]
Pretty much what the title says

**Comment by Mustard2** [comment]
This is because most of the properties in Blender has a max/min to infinity themselves, so it's transferred to the UI properties.

Or is there a property that has different max/min and it's transferred uncorrectly in the UI? If so, that might be a bug

**Comment by J4ke6599** [comment]
I noticed that custom properties imported to the UI will have the min max value set to I think 2 million. Any other node that uses floats seem to import fine.

**Comment by Mustard2** [comment]
Yeah I guess it's taking the value set on the property, I checked. Unfortunately I prefer this option rather than setting a random -1/1 or 0/1 without caring about the existent value.

Did you find any specific part where this evidently an error?

**Comment by J4ke6599** [comment]
Alright that's fine. It just would be nice to not have it set to some absurd number every time I add a float-point property to the UI.

**Comment by mokujinh** [comment]
Dunno if it is of any help, but a work around this issue is to assign the old "MustardUI - Float/Bool" on the name of the setting that is gonna be used on the UI and let the UI detection insert that into the UI on it's own,  for example, for a shader node: (MustardUI Float - Subsurface).  
That way all the values with these names (being shader nodes or shapekeys that use the "float" setting) get dettected with a default 0.000 min and 1.000 max values. But sadly the UI only detects it on the main mesh (body) itself, not in the extra meshes that could be outfits for example, those but if those settings will be linked to the ones in the main mesh in the end that is still useful.

**Comment by Mustard2** [comment]
@J4ke6599 I understand the issue, maybe I can add a sort of limiter when -inf + inf is available. But again, this is how it was intended in Blender when you add some king of values to the UI. :(

**Comment by Mustard2** [comment]
I was thinking about adding an option (False by default) to make people set a "default" max or min when adding custom properties, if the value is -inf/+inf. What do you think?

**Comment by Mustard2** [comment]
Closing as this does not seem to be of interest anymore. Please open a new Issue if you want to propose this change again. :)

**Comment by Mustard2** [comment]
Adding this option in 2026.9.
